docs/download_files.md

```diff
@@ -0,0 +1,69 @@
+# Downloadable files
+
+The ScPCA portal download packages include gene expression data, a QC report, and associated metadata for each processed sample.
+These files are delivered as a zip file.
+The folder structure within the zip file is determined by selecting to download all samples associated with a project or selecting individual samples.  
+
+## Download folder structure for project downloads: 
+_add in illustration of file structure for project_ 
+
+## Download folder structure for individual sample downloads: 
+_add in illustration of file structure for sample downloads_ 
+
+Sample folders (indicated by the `SCPCS` prefix) contain the files for all libraries (`SCPCL` prefix) derived from that biological sample. 
+Most samples only have one library that has been sequenced. 
+See {ref}`the difference between samples and libraries <faq:what is the difference between a sample and a library?>` for more information.
+
+The files associated with each library are:
+- An unfiltered counts file: `SCPCL000000_unfiltered.rds`, 
+- A filtered counts file: `SCPCL000000_filtered.rds`, 
+- A quality control report: `SCPCL000000_qc.html`, 
+- A metadata file: `SCPCL000000_metadata.json`. 
+Every download also includes a `libraries_metadata.csv` file containing metadata associated with each library included in the download.
```

Comment:
Trying to clarify here that this is for multiple libraries, not one per library. 

```suggestion
Every download also includes a single `libraries_metadata.csv` file containing metadata for all libraries included in the download.
```